.github/workflows/run_examples_snake_oil.yml

```diff
@@ -0,0 +1,69 @@
+name: Run Snake OIL demo
+
+on:
+ push:
+   branches:
+     - main
+     - 'version-**'
+ pull_request:
+
+jobs:
+  run-ert3-snakeoil:
+    timeout-minutes: 20
+    services:
+      postgres:
+        image: postgres:10.8
+        env:
+          POSTGRES_USER: postgres
```

Comment:
Why do we need these env variables?

Reply:
This is a copy from the analogue polynomial github action setup. But running snake_oil also through postgres is probably not necessary, so it can be skipped for snake_oil.